ui/job-view/details/tabs/SideBySide.jsx

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faExternalLinkAlt,
+  faSpinner,
+} from '@fortawesome/free-solid-svg-icons';
+import { Table } from 'reactstrap';
+
+import { getJobsUrl } from '../../../helpers/url';
+import { notify } from '../../redux/stores/notifications';
+import { getData } from '../../../helpers/http';
+import Clipboard from '../../../shared/Clipboard';
+
+import SideBySideVideo from './SideBySideVideo';
+
+class SideBySide extends React.PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sideBySideLoading: false,
+      sideBySideParams: undefined,
+    };
+  }
+
+  componentDidMount() {
+    const { sideBySideParams } = this.state;
+    if (!sideBySideParams) {
+      this.getSideBySideParams();
+    }
+  }
+
+  getSideBySideParams() {
+    const { jobDetails } = this.props;
+
+    // sideBySideParams example
+    // {
+    //   test_name: 'browsertime-tp6-essential-firefox-amazon',
+    //   new_test_name: null,
+    //   base_revision: 'fcc6a3eb9c660cce5574084c2b01521022a30d1f',
+    //   new_revision: 'cb9000cac930a412240edae67ceff231ec13018a',
+    //   base_branch: 'try',
+    //   new_branch: 'try',
+    //   platform: 'test-linux1804-64-shippable-qr',
+    //   new_platform: null,
+    //   overwrite: false,
+    //   cold: false,
+    //   warm: false,
+    //   most_similar: false,
+    //   search_crons: false,
+    //   skip_download: false,
+    //   output: null,
+    //   metric: 'speedindex',
+    //   vismetPath: false,
+    //   original: false,
+    //   skip_slow_gif: false,
+    // }
```

Comment:
do we still need these?

Reply:
I'd like to keep it

Reply:
Maybe we could leave an explanation comment on why we need to keep it